automation/test-linux.sh

```diff
@@ -63,48 +69,56 @@ fi
 
 delete_vm(){
   vm_name=$1
-  local template_name=$2
+
   set +e
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc process $2 -n $namespace -o json NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
```

Comment:
I would recommend you keep the local variables in shell functions instead of directly referencing `$2` in the command. Makes it easier to follow and maintain.

Reply:
Added the local variable